azkaban-web-server/src/main/java/cloudflow/servlets/ExecutionServlet.java

```diff
@@ -132,9 +122,95 @@ protected void handleGet(HttpServletRequest req, HttpServletResponse resp, Sessi
     return map;
   }
 
+  @Override
+  protected void handleGet(HttpServletRequest req, HttpServletResponse resp, Session session)
+      throws IOException {
+    Map<String, String> templateVariableMap = new HashMap<>();
+    String requestUri = req.getRequestURI();
+
+    if (GET_ALL_EXECUTIONS_URI.match(requestUri, templateVariableMap)) {
+      handleAllExecutions(req, resp);
+    } else if (GET_SINGLE_EXECUTION_URI.match(requestUri, templateVariableMap)) {
+      handleSingleExecution(req, resp);
+    } else if (GET_JOB_EXECUTION_INFO_URI.match(requestUri, templateVariableMap)) {
+      handleExecutionWithJobInfo(req, resp, session, templateVariableMap);
+    } else {
+      sendErrorResponse(resp, HttpServletResponse.SC_NOT_FOUND, DEFAULT_404_ERROR_MESSAGE);
+    }
+  }
+
   @Override
   protected void handlePost(HttpServletRequest req, HttpServletResponse resp, Session session)
-      throws ServletException, IOException {
+      throws IOException {
+
+    if (GET_ALL_EXECUTIONS_URI.match(req.getRequestURI(), new HashMap<>())) {
+      ExecutionParameters executionParameters = extractExecutionParameters(req, resp, session);
+      if (executionParameters == null) {
+        return;
+      }
+
+      String executionId;
+      try {
+        executionId = executionService.createExecution(executionParameters);
+      } catch (CloudFlowValidationException e) {
+        sendErrorResponse(resp, HttpServletResponse.SC_BAD_REQUEST, e.getMessage());
+        return;
+      } catch (CloudFlowException e) {
+        sendErrorResponse(resp, HttpServletResponse.SC_INTERNAL_SERVER_ERROR,
+            DEFAULT_500_ERROR_MESSAGE);
+        return;
+      }
+      logger.info(String.format("New execution of flow %s was successfully queued with id %s",
+          executionParameters.getFlowId(), executionId));
+      resp.setHeader("Location", GET_SINGLE_EXECUTION_URI.createURI(executionId));
```

Comment:
Nice! Didn't know this was supported.
Is it common to pass such information in the header ?

Reply:
when new resources are creating yes, it's common.